extensions/observability-devservices/testcontainers/src/main/java/io/quarkus/observability/testcontainers/LgtmContainer.java

```diff
@@ -25,12 +43,29 @@ public LgtmContainer(LgtmConfig config) {
         super(config);
         addExposedPorts(config.otlpPort());
         withLogConsumer(new LgtmContainerLogConsumer(log).withPrefix("LGTM"));
+        withCopyFileToContainer(
```

Comment:
This things should be made configurable -- at least: enabled / disabled.
e.g. you don't need scraping always, some registry impls know how to periodically push metrics

Reply:
Yes and it can be done automatically.
You can refactor things a bit so you can use his data from the ExtensionsCatalog object: https://github.com/quarkusio/quarkus/blob/7a4be43d5c0a56951dc864293413331ba9b04a34/extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/ObservabilityDevServiceProcessor.java#L120

Reply:
OK, this can be used with actual DevServices, but not with the other 2 "flavours" - DevResource and QuarkusTestResource.
Where DevResource is pretty much non-configurable, so that might be a problem.
Where QuarkusTestResource can probably get this config via QuarkusTestResourceLifecycleManager::init ...

Reply:
I will be the first to admit i am no expert on this stuff so any help is appreciated.  I basically wanted to show how we can load our own dashboard and get metrics from the running DEV mode as currenlty the LGTM defaults have dashboards that do nothing?

Reply:
No worries. 
I think we should probably show the minimum common denominator metrics. The ones common to both the Prometheus and the otlp registries.
This way, all metrics will show something and it's good enough to start with... It is also simpler to develop and it's just a meter of removing stuff from the dashboard.

Reply:
Oh OK if you tell me which sections you want to "stay" i can rip all others out.

Reply:
Will come up with something tomorrow

Reply:
Please, see my commit and comment bellow